### ISSUE TITLE
推理结果文件中有<img_-100>

### ISSUE THREAD

**Issue by sanwei111** [THUDM/ChatGLM-6B#1171]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

{"labels": "<image_-100><image_-100><image_-100><image_-100><image_-100><image_-100><image_-100><image_-100>xxxxxxxxxxxxxxxxxxxx", "predict": "xxxxxxxxxxxxxxxxxx"}

<image_-100>是什么东西？

### Steps To Reproduce

ef

### Environment

```markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :
efg
```

**Comment by yqkgithup** [comment]
捞一条，我也遇到了，使用v1.1.0模型的labels会出现这些字符，有大佬说改用v0.1.0模型就可以了，可能是v1.1.0的tokenizer的问题，求官方回答一下这些字符是否影响finetune和推理结果

**Comment by huilinbo** [comment]
我也遇到这个问题。求官方解答。

**Comment by lhy101** [comment]
同问

**Comment by xiningnlp** [comment]
修改tokenization_chatglm.py，最后一行：

```python

def _decode(
            self,
            token_ids: Union[int, List[int]],
            **kwargs
    ) -> str:
        if isinstance(token_ids, int):
            token_ids = [token_ids]
        if len(token_ids) == 0:
            return ""
        if self.pad_token_id in token_ids:  # remove pad
            token_ids = list(filter((self.pad_token_id).__ne__, token_ids))
        return self.sp_tokenizer.decode(token_ids)

```

这些小细节估计原作者也懒得管了，还是得自己看代码。
